content/03.results.md

```diff
@@ -195,6 +195,8 @@ This trend may have resulted from therapy-induced mutations that produced additi
 
 ### Transcriptomic Landscape of Pediatric Brain Tumors
 
+Since the majority of RNA-Seq samples in the PBTA cohort were prepared with ribosomal RNA depletion followed by stranded sequencing (N = 977) compared to those with poly-A selection (N = 58), the following analyses were performed using stranded samples only.
```

Comment:
We might expand this a tad to state plainly that a reason we are only using stranded is because the diagnoses are not evenly distributed across library preps so correcting is a challenge. Can refer to S7A probably as well (the polya/stranded count barplot).

Reply:
Updated in [f16f692](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/438/commits/f16f6929559a96c711afbacb5898d42dbf0514ff) - see if that looks OK to you - trying not to add too much to results at this point :)

Reply:
This is good! I can usually find more places to cut words when we close to the end of revisions too as needed.